internal/testfixtures/validating.go

```diff
@@ -209,20 +209,16 @@ func (vrwt validatingReadWriteTransaction) DeleteNamespaces(ctx context.Context,
 	return vrwt.delegate.DeleteNamespaces(ctx, nsNames...)
 }
 
-func (vrwt validatingReadWriteTransaction) WriteRelationships(ctx context.Context, mutations []*core.RelationTupleUpdate) error {
+func (vrwt validatingReadWriteTransaction) WriteRelationships(ctx context.Context, mutations []tuple.RelationshipUpdate) error {
 	if err := validateUpdatesToWrite(mutations...); err != nil {
 		return err
 	}
 
 	// Ensure there are no duplicate mutations.
 	tupleSet := mapz.NewSet[string]()
 	for _, mutation := range mutations {
-		if err := mutation.Validate(); err != nil {
-			return err
-		}
-
-		if !tupleSet.Add(tuple.StringWithoutCaveat(mutation.Tuple)) {
-			return fmt.Errorf("found duplicate update for relationship %s", tuple.StringWithoutCaveat(mutation.Tuple))
+		if !tupleSet.Add(tuple.StringWithoutCaveat(mutation.Relationship)) {
```

Comment:
was the removed validation call redundant?

Reply:
No, but now validation occurs at the translation point